package.json

```diff
@@ -4,48 +4,14 @@
   "description": "Provides a common starting point for all Element blocks",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
-    "build": "npm test && rollup -c rollup.config.prod.js",
-    "build:no-test": "rollup -c rollup.config.prod.js",
-    "start": "rollup -c rollup.config.dev.js --watch"
+    "test": "block-scripts test",
+    "build": "block-scripts build",
+    "start": "block-scripts start"
   },
   "author": "Volusion, LLC",
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "@volusion/element-components": "*",
-    "@volusion/element-proptypes": "*",
-    "aphrodite": "^2.2.3",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.3",
-    "babel-plugin-external-helpers": "^6.22.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-latest": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1",
-    "enzyme": "^3.6.0",
-    "enzyme-adapter-react-16": "^1.5.0",
-    "enzyme-to-json": "^3.3.4",
-    "eslint": "^4.19.1",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-config-standard-react": "^6.0.0",
-    "eslint-plugin-prettier": "^2.6.0",
-    "eslint-plugin-react": "^7.8.2",
-    "get-port": "^4.0.0",
-    "jest": "^23.6.0",
-    "jest-aphrodite-react": "^2.2.0",
-    "jest-runner-eslint": "^0.6.0",
-    "prettier": "^1.13.3",
-    "react": "^16.5.2",
-    "react-dom": "^16.5.2",
-    "rollup": "^0.59.2",
-    "rollup-plugin-babel": "^3.0.4",
-    "rollup-plugin-commonjs": "^9.1.3",
-    "rollup-plugin-eslint": "^4.0.0",
-    "rollup-plugin-livereload": "^0.6.0",
-    "rollup-plugin-node-resolve": "^3.3.0",
-    "rollup-plugin-replace": "^2.0.0",
-    "rollup-plugin-serve": "^0.6.0",
-    "rollup-plugin-uglify": "^4.0.0"
+    "@volusion/block-scripts": "*"
```

Comment:
I'd like to see `element` in this module name somewhere.

Reply:
`@volusion/element-block-scripts`?